torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -395,7 +395,7 @@ static TensorView* newForReduction(
 
 TensorView* reductionOp(
     BinaryOpType reduction_op_type,
-    const std::vector<int>& axes,
+    const std::vector<int64_t>& axes,
```

Comment:
Why are the axes variables being increased from 4 bytes to 8 bytes?  I thought we would only ever have 8+1 positions?  I am guessing there is some mechanical or legality issue with narrowing.

Reply:
reverted.